packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -317,6 +351,74 @@ void sendInitialized() {
     }
   }
 
+
+  List<Messages.GetEmbeddedSubtitlesMessage> getEmbeddedSubtitles() {
+    List<Messages.GetEmbeddedSubtitlesMessage> subtitleItems = new ArrayList<>();
+    TrackGroupArray trackGroups;
+    int rendererIndex = 2;
```

Comment:
It is my understanding that `C.TRACK_TYPE_VIDEO` [maps to 2](https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/C.html#TRACK_TYPE_VIDEO), which is why you are using it here. Can you use that constant for clarity?